Fix for fancy checkbox in firefox + improving main content css + add page templates for docs site

bitterdev opened the pull request:
While cross browser testing the theme i have seen that the checkboxes are not rendered. This pr is fixing that issue.

aembler commented:
Can you pull down a copy of master and redo this pull request? It has merge conflicts and I assume it's because you need to do a git pull or rebase first

bitterdev commented:
Not this should work fine.

aembler commented on data.xml:
```diff
@@ -13,7 +13,129 @@
             <type placeholder=""/>
         </attributekey>
-
+        <attributekey handle="profile_private_messages_enabled" name="Private Messages" package="concrete_cms_theme"
```
I don't think these attributekeys should be here. Shouldn't this have been moved into the community package?

bitterdev replied:
Okay i will move this to the community package.

bitterdev commented:
Removed this section. No need to copy this to the community repo because it's already there (see data.xml)
I have also added some css rules that are making the main content looking nicer (useful for the docs site)

bitterdev commented:
Added page templates for documentation site
